modules/cnv_connecting_resource_bridge_network.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_attaching_resource_multiple_networks.adoc
+
+[id="connecting-resource-bridge-network-{context}"]
+= Connecting a resource to a bridge network
```

Comment:
English: would it not be nicer to say

> bridge-based network?

Reply:
Not sure which sounds better, to be honest. I will change it to bridge-based network though, because that seems more accurate.